### ISSUE TITLE
Не ищет прокси сервер

### ISSUE THREAD

**Issue by glaptize** [zxcnoname666/SoundCloud-Desktop#10]
![soundcloud](https://github.com/user-attachments/assets/fd03163d-6d4d-4b25-8552-051b5054acf6)

Я видел что у чела была такая же проблема.Без впна на soundcloud не заходит.

**Comment by zxcloli666** [comment]
Привет.
Попробуй последнюю версию установить, я в ней отключил tls1.3, поэтому проблемы быть не должно.

**Comment by glaptize** [comment]
Всё также, проблема не исчезла

**Comment by zxcloli666** [comment]
У тебя какой провайдер?

**Comment by glaptize** [comment]
Билайн

**Comment by zxcloli666** [comment]
У них правила блокировки более жесткие.
Я чуть позже как можно трафик зашифровать, потому что у тебя провайдер вообще все подключения проверяет, из-за этого прокси не работают

**Comment by glaptize** [comment]
Ждать обновлений?

**Comment by zxcloli666** [comment]
Да. Сейчас добавлю еще shadowsocks прокси, может с ним заработает

**Comment by zxcloli666** [comment]
_В архиве app.asar, распакуй его в удобную папку, гитхаб не позволяет загрузить asar файл вне архива._

Попробуй установить этот app.asar в папку, заменив текущий app.asar:
%APPDATA%/../Local/Programs/soundcloud/resources

Я туда добавил socks proxy, но без обфускации трафика

[app.zip](https://github.com/user-attachments/files/18156727/app.zip)

**Comment by glaptize** [comment]
Да, всё работает, я спокойно вошёл в аккаунт и могу слушать треки.
Спасибо.

**Comment by zxcloli666** [comment]
Пожалуйста
